implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -176,6 +176,8 @@ impl NodeManager {
             .await?;
         self.start_authenticated_service_impl(ctx, "authenticated".into())
             .await?;
+        self.create_secure_channel_listener_impl(ctx.address(), None)
+            .await?;
```

Comment:
This didn't work and hanged for me at `node create`
It needs the address "api"

This worked:

```rust
        self.create_secure_channel_listener_impl("api".into(), None)
            .await?;
```

Reply:
Ahh, my bad. `ctx.address()` returns primary worker's address, I should have looked properly.